mitodl/micromasters
# Removed fields from homepage object and CMS

#### What are the relevant tickets?
- Fixes https://github.com/mitodl/micromasters/issues/1090

#### What's this PR do?
- Removed Title from cms and template
- Title Background from cms and template 
- Create migration

#### Where should the reviewer start?
- cms/models.py

#### How should this be manually tested?
- See /cms/  => Explorer
- see home page

@aliceriot @pdpinch 

#### Screenshots (if appropriate)
<img width="1279" alt="screen shot 2016-09-29 at 4 25 31 pm" src="https://cloud.githubusercontent.com/assets/10431250/18951991/68259b0c-8661-11e6-8dfe-4940672c650f.png">
![screen shot 2016-09-29 at 4 25 43 pm](https://cloud.githubusercontent.com/assets/10431250/18952006/783cbad4-8661-11e6-8577-3f5e724ad73d.png)
